layouts/shortcodes/djm-install-troubleshooting.en.md

```diff
@@ -0,0 +1,10 @@
+Data Jobs Monitoring requires to have the Datadog Agent running in the background. You can check that it is correctly installed and running on your cluster with this command:
+  ```shell
+  sudo datadog-agent status
+  ```
+If it is not the case, you may want to check the log file of the installation. On your cluster, these logs are located in `/tmp/datadog-djm-init.log`.
```

Comment:
```suggestion
If there is no Agent running, check the log file of the installation. On your cluster, these logs are located in `/tmp/datadog-djm-init.log`.
```

Reply:
Hi Heston, I'm sorry I had unpushed changes suggesed by my team. I just pushed them, can you take another look please ?